java-frontend/src/main/java/org/sonar/java/ast/JavaAstScanner.java

```diff
@@ -66,15 +63,15 @@ public void scan(Iterable<File> files) {
           break;
         }
         simpleScan(file);
-        progressReport.nextFile();
+//        progressReport.nextFile();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACommentedOutCodeLine)